code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1444,6 +1446,12 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				user.visible_message("<span class='danger'>[user.name] shoves [target.name] into \the [target_disposal_bin]!</span>",
 					"<span class='danger'>You shove [target.name] into \the [target_disposal_bin]!</span>", null, COMBAT_MESSAGE_RANGE)
 				log_combat(user, target, "shoved", "into [target_disposal_bin] (disposal bin)")
+			else if(target_pool)
```

Comment:
Out of scope but this is starting to seem like a switch might be more efficient for this check and the three above.

Reply:
Unless the main thing was changed to have a varibable of what you are being shoved into, a switch statement wouldn't really work here because of the changing variables

Reply:
Wait youc an shove people into pool now? BASED?

Reply:
Yea